csi.proto

```diff
@@ -1163,6 +1196,15 @@ message Snapshot {
   // `volume_content_source` in a `CreateVolumeRequest`. The default
   // value is false. This field is REQUIRED.
   bool ready_to_use = 5;
+
+  // The ID of the volume group snapshot that this snapshot is part of.
+  // It uniquely identifies the group snapshot on the storage system.
+  // This field is OPTIONAL.
+  // If this snapshot is a member of the volume group snapshot, the SP
+  // SHOULD provide the ID of the volume group snapshot in this field.
+  // If provided, CO MUST use this field to indicate that this snapshot
+  // is part of the specified group snapshot.
+  string group_snapshot_id = 6 [(alpha_field) = true];
```

Comment:
If this message is inside a VolumeGroupSnapshot message, the value MUST be the same as the group_snapshot_id in that message.

Reply:
Updated.